src/Flux.js

```diff
@@ -141,6 +123,26 @@ export default class Flux extends EventEmitter {
     return actionIds;
   }
 
+  performAction(actionId, action, ...actionArgs) {
+    const body = action.apply(this, actionArgs);
```

Comment:
This should be `const body = action.apply(null, actionArgs);`. `action` is already bound to the action creator. We should make clear that there is no additional binding going on here.